libs/application/templates/parental-leave/src/fields/OtherParent/index.tsx

```diff
@@ -8,38 +8,126 @@ import {
 import { useLocale } from '@island.is/localization'
 import { Box, Text } from '@island.is/island-ui/core'
 import { RadioFormField } from '@island.is/application/ui-fields'
-import { getOtherParentOptions } from '../../lib/parentalLeaveUtils'
-import { SPOUSE, NO, SINGLE } from '../../constants'
+import {
+  getApplicationAnswers,
+  getOtherParentOptions,
+} from '../../lib/parentalLeaveUtils'
+import { SPOUSE, NO, SINGLE, TransferRightsOption } from '../../constants'
 import { useFormContext } from 'react-hook-form'
 
 export const OtherParent: FC<FieldBaseProps> = ({ application, field }) => {
   const { id, title } = field
+  const { errors, setValue, register } = useFormContext()
   const { formatMessage } = useLocale()
-  const { errors, setValue } = useFormContext()
+
+  // const {
```

Comment:
This commented out code, can be removed.

Reply:
I have them there so if someone want to take this feature could continue to work with it